src/pages/CountryPage.js

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import StateList from '../components/StateList';
+import CountryData from '../components/CountryData';
+import { closeHamburger } from '../redux/FunctionalitySlice';
+
+const CountryPage = () => {
+  const dispatch = useDispatch();
+  return (
+    <>
+      <div className="backToHomepage">
+        <Link to="/" onClick={() => dispatch(closeHamburger())}>Back to homepage</Link>
```

Comment:
- [ ] The filter functionality is not working on the individual (country) pages, kindly fix this so it works as intended. I guess the issue is that you're not generating a unique id (key) for each city

- The console error:
![Screenshot_20220728_075409](https://user-images.githubusercontent.com/25227022/181442384-57701c01-0e8a-49be-ac72-23649b0ae365.png)

Reply:
Hey, @Ismailco Can you specify this? I couldn't reproduce the error.

Reply:
@Gopxfs, I checked again and it's working normally, I don't understand what happened, but sometimes the same key is used for different cities, in this case, it breaks the filter function